docs/pages/guides/migrate-to-outpost.mdx

```diff
@@ -0,0 +1,216 @@
+
+### For Event Consumers:
+
+- **Streamlined infrastructure & operations**
+
+  Eliminate the need for API gateways, load balancers, HTTP consumers, and other infrastructure components, reducing maintenance overhead.
+
+- **Reduced developer burden**
+
+  Receive events directly to existing or preferred infrastructure and make use of existing and familiar ecosystem tooling.
+
+- **Predictable behavior**
+
+  Standardize event expectations—the message bus handles timeouts, retries, and security.
+
+## How to Migrate to Outpost
+
+The following sections outline the key considerations when migrating webhook infrastructure to Outpost.
+
+### Terminology
+
+Outpost has the following concepts that you need to map to your existing webhook infrastructure provider:
+
+- **Tenants**: A tenant represents a customer organization.
+- **Destination Types**: The type of destination where events will be delivered. For example, webhook, Hookdeck Event Gateway, or AWS SQS.
+- **Destinations**: A subscription configured to deliver the event to a destination type. For example, a webhook destination with a specific URL.
+- **Events**: An event is a piece of data that represents an action that occurred in your system. For example, a user signed up or a payment was processed.
+- **Topics**: A topic is a way to categorize events and is a common concept found in Pub/Sub messaging. For example, a `user.created`.
+
+### Webhook Event Delivery Method
+
+Webhooks are delivered via HTTP `POST` requests.
+
+### Webhook Event HTTP Headers
+
+- `x-outpost-event-id`: A unique identifier for the event within the Outpost installation
+- `x-outpost-signature`: The signature generated using the webhook "t=1741787378,v0=86245a49c99fac7f30c616a29530c49cee6291293b38453488b3feb99618ee22",
+- `x-outpost-timestamp`: A Unix timestep representing the time the event was generated
+- `x-outpost-topic`: The topic of the event. For example, `user.created`.
+
+The `x-outpost` prefix can be customized. See [Webhook Customization](#webhook-customization).
+
+### Webhook Event Payload Format
+
+Outpost events are JSON payloads.
+
+### Webhook Event Payload Structure
+
+Outpost does not enforce a specific event payload structure. You can define the structure of the event payload as needed. However, the content must be JSON.
+
+### Authentication
+
+By default, Outpost uses a secret key to sign the webhook payload. The secret key is used to generate a signature that is included in the `x-outpost-signature` header. The signature is generated using the HMAC-SHA256 algorithm with Hex encoding.
+
+The `x-outpost` prefix can be customized. See [Webhook Customization](#webhook-customization).
+
+### Webhook Customization
+
+You can customize the follow webhook features and behaviors. The full path to the YAML configuration is prefixed with `outpost.config.destinations.webhook`.
```

Comment:
I take from this `outpost.config.destinations.webhook` that you base this off of the Helm YAML config thingy.

Just to clarify, Outpost config schema is without `outpost.config`, it's there because Helm involves other infra things beyond just Outpost config.

I'm pushing a commit to fix some typo and update this but figured I'd clarify in case you want to make changes after.